### PR TITLE
Add subset property UBERON:8410007 added_for_HCA

### PR DESCRIPTION
**Pull request by paolaroncaglia**
Addresses #1700

### CODE REVIEW

**Comment by paolaroncaglia** [comment]
@nicolevasilevsky or @dosumis 
Could one of you please review this pull request when you have a chance? So I can start using the tag in other pull requests :-) 
No hurry, and enjoy the holidays. Thanks!

File: uberon_edit.obo [hunk 1]
```diff
@@ -19,6 +19,7 @@ subsetdef: non_informative "abstract class brought in to group ontology classes
 subsetdef: organ_slim "organs, excluding individual muscles and skeletal elements"
 subsetdef: pheno_slim "Phenotype slim"
 subsetdef: phenotype_rcn "A subset specifically created for the 2012 Phenotype RCN meeting. Includes some human-specific terms that may eventually be removed when they are adequately represented with part of relationships in FMA"
+subsetdef: UBERON:8410007 "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
```
Comment (nicolevasilevsky) [inline uberon_edit.obo:22]: this doesn't seem to be formatted like the other subset defs - will UBERON:841007 become the label? Is this what you intended?

It's not letting me open this branch, so I can't look in Protege.

Comment (paolaroncaglia) [reply uberon_edit.obo:22]: @nicolevasilevsky 
> this doesn't seem to be formatted like the other subset defs - will UBERON:841007 become the label? Is this what you intended?
> It's not letting me open this branch, so I can't look in Protege.

Thanks Nicole - no, I meant "added_for_HCA" to be the label... let me try again!

Comment (paolaroncaglia) [reply uberon_edit.obo:22]: @nicolevasilevsky 
I tried again and see the same issue. I'll attach a screenshot of what it looks like in Protege for me:
Screen Shot 2021-01-13 at 15 31 07

Comment (dosumis) [inline uberon_edit.obo:22]: probably easiest to edit OBO file.  

subsetdef: UBERON:8410007 "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."
>
subsetdef:  added_for_HCA "Classes tagged with this subset property were added upon request from the Human Cell Atlas (HCA)."

**Review by dosumis — CHANGES_REQUESTED** [review]
See suggested fix in comment

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by paolaroncaglia** [comment]
@dosumis 
Could you approve and merge, please? Thanks.

**Review by dosumis — APPROVED** [review]
